NOTICE

```diff
@@ -207,7 +207,14 @@ Runtime dependencies:
     Developed by: Twitter, Inc.
     License name: MIT License
     Notice:       Copyright (c) 2011-2016 Twitter, Inc.
-  
+
+  ch.qos.logback:1.0.7
```

Comment:
This seems to be the same artifact as [line 218](https://github.com/apache/brooklyn/pull/35/files#diff-aa14599dc3fdfa9cc81a7c3c247b77b9R218) is that expected?